### ISSUE TITLE
Update check in 2.1.0 does not notify of new oficial update like 2.2.0

### ISSUE THREAD

**Issue by cesmed** [objective-see/LuLu#302]
Hi,
  I noticed that the update check fails in 2.1.0.

The UI shows this but there is a new release 2.2.0

![imagen](https://user-images.githubusercontent.com/2403969/104856438-efa84d80-5912-11eb-87b5-b5c8a771e58a.png)

I have MacOs Big Sur 11.1 on iMac 27'' (Late 2013)

Thanks for this great firewall.

**Comment by dannykorpan** [comment]
Same problem.

**Comment by Stellarspace** [comment]
Built-in updaters are usually unreliable.

**Comment by julienbordet** [comment]
Yes, same here for Lulu 2.0.0 on one of my Mac. (MacBook Pro 13' 2015, Intel)

**Comment by J-Prince** [comment]
MacOS 11.2, MacBook Pro (13-inch, 2020 Intel)
Same issue, manually pressing "Check Now" returns "Installed version 2.1.0 is the latest".

**Comment by mrc247-DE** [comment]
macOS 10.15.7
LuLu 1.2.3

same here, when I check for an update I get the message: "no new versions"

**Comment by kennymc-c** [comment]
Same problem with the 2.3.0 update

**Comment by denizt** [comment]
The update process uses a JSON file that is defined as a constant https://github.com/objective-see/LuLu/blob/45e0434e511c80b3710e52867f75567aa6e7f54e/LuLu/Shared/consts.h#L77

There is already a pull request that tries to mitigate this by using git tags however best would be to fix the release process and generate the file properly on release, the code for the website/release update scripts don't seem to be open source, so only @objective-see  can fix it at this stage I reckon.
